src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationSender.java

```diff
@@ -60,7 +61,7 @@ private void sendErrorMessageToQueue(
 
```

Comment:
Should be:
String messageId = StringUtils.joinWith("_", containerName, zipFilename);
This messageId is used in log.info(...)

Reply:
Fixed it